x/concentrated-liquidity/swaps_test.go

```diff
@@ -199,43 +188,41 @@ func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
 				_, _, _, err := s.App.ConcentratedLiquidityKeeper.CreatePosition(ctx, poolId, s.TestAccs[0], DefaultAmt0, DefaultAmt1, sdk.ZeroInt(), sdk.ZeroInt(), DefaultLowerTick, DefaultUpperTick)
 				s.Require().NoError(err)
 				// params
-				// liquidity (1st):  1517818840.967415409394235163
-				// sqrtPriceNext:    67.416477345120317059 which is 4544.98141762512095360 (this is calculated by finding the closest tick LTE the upper range of the first range) https://www.wolframalpha.com/input?key=&i2d=true&i=Power%5B1.0001%2CDivide%5B84222%2C2%5D%5D
+				// liquidity (1st):  1517882343.751510418088349649
+				// sqrtPriceNext:    67.416615162732695594 which is 4545
 				// sqrtPriceCurrent: 70.710678118654752440 which is 5000
-				// expectedTokenIn:  1048863.4367036217364698923810 rounded up https://www.wolframalpha.com/input?i=%281517818840.967415409394235163+*+%2870.710678118654752440+-+67.416477345120317059%29%29+%2F+%2867.416477345120317059+*+70.710678118654752440%29
-				// expectedTokenOut: 5000000000.000 rounded down https://www.wolframalpha.com/input?key=&i=1517818840.967415409394235163+*+%2870.710678118654752440-+67.416477345120317059%29
-				// expectedTick:     84222.0 rounded down https://www.wolframalpha.com/input?i2d=true&i=Log%5B1.0001%2C4544.98141762512095360%5D
+				// expectedTokenIn:  1048861.292545921016650926872369 rounded up https://-www.wolframalpha.com/input?i=%281517882343.751510418088349649+*+%2870.710678118654752440+-+67.416615162732695594%29%29+%2F+%2867.416615162732695594+*+70.710678118654752440%29
+				// expectedTokenOut: 5000000000.00000000000000 rounded down https://www.wolframalpha.com/input?i=1517882343.751510418088349649+*+%2870.710678118654752440-+67.416615162732695594%29
```

Comment:
I think this value is off but can be addressed here: https://github.com/osmosis-labs/osmosis/pull/3962